post.html

```diff
@@ -1,20 +1,26 @@
 <!DOCTYPE html>
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1" />
 <title>Reader Post</title>
 <style>
+  @import url("./index.css");
+  @import url("./common.css");
+  @import url("./theme.css");
+  @import url("./timeline.css");
   @import url("./post.css");
 </style>
-<div id="post-container">
-  <!-- https://staticpub.mauve.moe/helloworld.html -->
-  <!-- hyper://staticpub.mauve.moe/helloworld.html -->
-  <!-- ipfs://bafybeifslnipwp5uanmhkckokwuse7h5gfrrjzqq4jg5oxewxbzrdcdawu -->
-  <distributed-post id="distributedPost"></distributed-post>
+<div class="container">
+  <sidebar-nav></sidebar-nav>
+  <div class="individual-post">
+    <distributed-post id="distributedPost"></distributed-post>
+  </div>
+  <div class="right-column">
+    <!-- This is an empty column to balance the layout -->
+  </div>
 </div>
-
-<script type="module" src="./post.js"></script>
-<!-- Dynamically set post URL from query string parameter -->
-<script>
+<script type="module">
   const params = new URLSearchParams(window.location.search);
-  const postUrl = params.get("url");
+  const postUrl = params.get("url") || "ipns://hypha.coop/dripline/announcing-dp-social-inbox/create/1701734400.jsonld";
```

Comment:
```suggestion
  const postUrl = params.get("url") || "ipns://hypha.coop/dripline/announcing-dp-social-inbox.ipns.jsonld";
```

We need to link to the p2p specfic file for the post not just the create activity.